pkg/operation/botanist/component/kubeapiserver/dependency_watchdog_test.go

```diff
@@ -67,13 +68,45 @@ var _ = Describe("DependencyWatchdog", func() {
 					Internal:      &scalerapi.ProbeDetails{KubeconfigSecretName: "dependency-watchdog-internal-probe"},
 					PeriodSeconds: pointer.Int32(30),
 				},
-				DependantScales: []*scalerapi.DependantScaleDetails{{
-					ScaleRef: autoscalingv1.CrossVersionObjectReference{
-						APIVersion: "apps/v1",
-						Kind:       "Deployment",
-						Name:       "kube-controller-manager",
+				DependantScales: []*scalerapi.DependantScaleDetails{
+					{
+						ScaleRef: autoscalingv1.CrossVersionObjectReference{
+							APIVersion: appsv1.SchemeGroupVersion.String(),
+							Kind:       "Deployment",
+							Name:       "kube-controller-manager",
+						},
+						ScaleUpDelaySeconds: pointer.Int32(120),
+					},
+					{
+						ScaleRef: autoscalingv1.CrossVersionObjectReference{
+							APIVersion: appsv1.SchemeGroupVersion.String(),
```

Comment:
same here

Reply:
Sorry for that, it's fixed with https://github.com/gardener/gardener/commit/df84f18468db5fc5ffb1d37e11b9852013c217cc